inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/resources/.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+// http://eslint.org/docs/user-guide/configuring
```

Comment:
Is it important that this is a hidden file? I'm afraid people working on the project may miss it.

Reply:
According to the ESLint Configuration page this seems correct.
But the page states that it could also be added to a field `eslintConfig` within package.json file.